Gruntfile.js

```diff
@@ -177,80 +177,80 @@ module.exports = function(grunt) {
 
           // This is a list of all dependencies, mapped to their AMD identifier.
           paths: {
-            'app': 'src/app',
+            'app': 'app',
 
             // core
-            'canvas': 'src/core/canvas',
-            'constants': 'src/core/constants',
-            'core': 'src/core/core',
-            'environment': 'src/core/environment',
-            'helpers': 'src/core/error_helpers',
-            'p5.Element': 'src/core/p5.Element',
-            'p5.Graphics': 'src/core/p5.Graphics',
-            'p5.Graphics2D': 'src/core/p5.Graphics2D',
-            'rendering.rendering': 'src/core/rendering',
-            'shape.2d_primitives': 'src/core/2d_primitives',
-            'shape.attributes': 'src/core/attributes',
-            'shape.curves': 'src/core/curves',
-            'shape.vertex': 'src/core/vertex',
-            'shim': 'src/core/shim',
-            'structure': 'src/core/structure',
-            'transform': 'src/core/transform',
+            // 'canvas': 'core/canvas',
+            // 'constants': 'core/constants',
+            // 'core': 'core/core',
+            // 'environment': 'core/environment',
+            // 'helpers': 'core/error_helpers',
+            // 'p5.Element': 'core/p5.Element',
+            // 'p5.Graphics': 'core/p5.Graphics',
+            // 'p5.Graphics2D': 'core/p5.Graphics2D',
+            // 'rendering.rendering': 'core/rendering',
+            // 'shape.2d_primitives': 'core/2d_primitives',
+            // 'shape.attributes': 'core/attributes',
+            // 'shape.curves': 'core/curves',
+            // 'shape.vertex': 'core/vertex',
+            // 'shim': 'core/shim',
+            // 'structure': 'core/structure',
+            // 'transform': 'core/transform',
```

Comment:
I'm leaving these in, just commented-out, until this set of changes is "done", so that it's easier to visually assess how many modules are affected